docs/getting-started-guides/kubeadm.md

```diff
@@ -195,6 +195,38 @@ Once a pod network has been installed, you can confirm that it is working by che
 
 And once the `kube-dns` pod is up and running, you can continue by joining your nodes.
 
+If you see following status 
+```
+NAMESPACE     NAME                              READY     STATUS              RESTARTS   AGE
+kube-system   canal-node-f0lqp                  2/3       RunContainerError   2          48s
+```
+Or 
+
+```
+kube-system   canal-node-77d0h                  2/3       CrashLoopBackOff    3          3m
+kube-system   kube-dns-2924299975-7q1vq         0/4       ContainerCreating   0          15m
+```
+The three status ```RunContainerError``` and ```CrashLoopBackOff``` and ```ContainerCreating``` very common. 
+
+You may have trouble in configure. to diagnose what happened. you can using ```kubectl describe -n kube-system po {YOUR_POD_NAME}``` to check what's in logs. do not using kubectl logs. you will got
```

Comment:
Grammar: 

**D**o not **use** kubectl logs; you will **get the following error:**